helm/prometheus-rules/templates/platform/atlas/alerting-rules/statefulset.management-cluster.rules.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+{{- if not .Values.mimir.enabled }}
+    cluster_type: "management_cluster"
+{{- end }}
+  name: deployment.management-cluster.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: statefulset
+    rules:
+    - alert: StatefulsetNotSatisfiedAtlas
+      annotations:
+        description: '{{`Statefulset {{ $labels.namespace}}/{{ $labels.statefulset }} is not satisfied.`}}'
+        opsrecipe: deployment-not-satisfied/
+      expr: |-
+        kube_statefulset_status_replicas{cluster_type="management_cluster", statefulset=~"loki.*|mimir.*"}
```

Comment:
Maybe we use namespace instead lf statefulset? The values would be the same but we would not need to use regex?

Reply:
I wanted to stick to the current `deploymentnotsatisfied` alert's behaviour.

I guess the reason why it's based on the pod name is for apps running in `kube-system`, where we have to filter by pod name to then route to the proper team.

So, for today/current case, filtering by namespace would work, but if we want to add a kube-system statefulset to the alert we would have to revert.

Reply:
Sure :)